pkg/virt-controller/watch/vm.go

```diff
@@ -2779,30 +2886,13 @@ func (c *VMController) setupLiveFeatures(
 	vm *virtv1.VirtualMachine,
 	vmi, VMIDefaults *virtv1.VirtualMachineInstance) {
 	const (
-		maxSocketsRatio = 4
+		maxRatio = 4
```

Comment:
Following our conversation about CPU HP & instanetypes, we would need to move the constant to virt-config

Reply:
Sure can do.

Reply:
Done 7d964c60dc549249c09d53bee265b1fed0b09f7f